src/Jellyfin.Plugin.Dlna/Api/DlnaServerController.cs

```diff
@@ -66,7 +66,12 @@ public ActionResult<string> GetDescriptionXml([FromRoute, Required] string serve
         string? userAgent = Request.Headers.UserAgent;
         if (userAgent is not null)
         {
-            userAgent = userAgent.Substring(0, userAgent.IndexOf('/'));
+            var firstIndexOfSlash = userAgent.IndexOf('/');
+            if (firstIndexOfSlash > 0)
+            {
+                userAgent = userAgent.Substring(0, firstIndexOfSlash);
```

Comment:
Can this be changed to just use our `LeftPart` function instead?

Reply:
I don't think this is available in a plugin 🤔

Reply:
Not sure if it's worth pulling in that package just to convert a string to a span, use the extension function and then convert back to a string